generator.js

```diff
@@ -11,6 +11,8 @@ const ICON_FAMILIES = [
   { id: 'materialiconstwotone', postfix: 'TwoTone' },
 ]
 
+const ignoredIcons = ['addchart']
```

Comment:
Wdyt about adding a comment why we ignore this icon? Could be helpful in the future.

Reply:
Great idea, thanks.